calculateSNR.m

```diff
@@ -264,20 +263,40 @@
         avgBold = mean(allRunsBoldDT, 3);
         avgRawBold = mean(allRunsBoldRaw, 3);
 
+        %% run FFT on average bold
         % ----------------------------------------------------------------
-        % zscore at target frequency
-        % run FFT on average bold
-        [AvrZTarget, cfg, FT] = calculateFourier(avgBold, avgRawBold, cfg);
-
+        % Zscore at target frequency. 
+        % (1) calculate magnitude specta using FFT
+        % (2) exctract magnitude at the target frequency bin -> this is
+        % "signal"
+        % (3) extract magnitudes at surrounding frequency bins -> this is noise
+        % (4) calculate zscore as (signal-mean(noise)) / std(noise)
+        % do this for each voxel and return in array AvgZTarget
+        [AvgZTarget, cfg, FT] = calculateFourier(avgBold, avgRawBold, cfg);
+        
```

Comment:
I moved the explanation up, because explanation is on how/what calculateFourier doing. No?